deployments/jwebbinar/config/common.yaml

```diff
@@ -21,7 +21,7 @@ jupyterhub:
         c.KubeSpawner.lifecycle_hooks = {
             "postStart": {
                 "exec": {
-                    "command": ["/bin/sh", "-c", "rsync -r /etc/default-home-contents/ /home/jovyan; /opt/environments/setup-notebooks; chmod +x /home/jovyan/Desktop/*.desktop; /opt/common-scripts/symlink-crds; /opt/environments/remove-kernel jdaviz"]
+                    "command": ["/bin/sh", "-c", "rsync -r /etc/default-home-contents/ /home/jovyan; /opt/environments/setup-notebooks; chmod +x /home/jovyan/Desktop/*.desktop; /opt/common-scripts/symlink-crds; /opt/environments/remove-kernel jdaviz; rm -rf /home/jovyan/.user-dirs/Desktop"]
```

Comment:
@jaytmiller - note the removal of `/home/jovyan/.user-dirs/Desktop` at the end.  This should be generalized somehow.  

I've been wondering if we should have a common command string or something that we run here, along with mission specific stuff...we've acquired all this cruft that is really only necessary to keep existing users' environments clean, and I'm betting there will be more as time goes on.

Reply:
I think all the cruft should be factored out into environments/post-start-hook and written as normal shell script(s).   That script should have two modes depending on whether or not the image is running on the hub;  In the common Dockerfile.trailer the script gets called with a "docker" parameter to set up the built-in $HOME for standalone runs,  in the config YAML it is called with no parameter for running on the hub with AWS services available.   Once the script is out of the list syntax into normal bash it's easier to deal with.    We could toss in a script of common things if needed which can be called by the mission specific post-start-hook scripts.